src/uniparc/config/UniParcFacetConfiguration.tsx

```diff
@@ -1,6 +1,6 @@
 export enum FacetsEnum {
   OrganismName = 'organism_name',
-  Database = 'database',
+  Database = 'database_facet',
```

Comment:
I think that was the only actual change?

Reply:
Yeah, that's it really. Just more data in the mock response but it's all the same type.